cfe_internal/enterprise/federation/federation.cf

```diff
@@ -153,12 +153,20 @@ bundle agent transport_user
           "$(ssh_known_hosts)",
           "$(ssh_config)"
         };
+      "ssh_home_cil_file" string => "/tmp/cftransport_ssh_home_t.cil";
+
   classes:
     enabled::
-      "selinux_enabled"
-        expression => strcmp("Enforcing", execresult("getenforce", useshell));
-      "missing_ssh_context"
-        expression => not( or(
+      _stdlib_path_exists_getenforce::
+        "selinux_enabled"
+          expression => strcmp("Enforcing", execresult("$(paths.getenforce)", useshell));
+
+      selinux_enabled::
+        "fix_ssh_context"
```

Comment:
I think we should use use a class that describes the state, not the desired action. Ssh_context_missing for example. That makes the context of the command more clear. When selinux is enabled and there is a missing context command should run.... Instead of when selinux is enabled and I should fix ssh context run command.

Reply:
Perfect. Makes sense. Will change! :+1: